.github/workflows/update-deps.yml

```diff
@@ -11,6 +11,7 @@ on:
 
 jobs:
   cli:
+    if: github.repository == 'getsentry/sentry-android-gradle-plugin'
```

Comment:
Disable updating deps jobs in forked repo, see the error in https://github.com/Goooler/sentry-android-gradle-plugin/actions/runs/4592071451/jobs/8108791057